### ISSUE TITLE
Scraped content is different from authenticated view of Facebook page

### ISSUE THREAD

**Issue by mossmoss** [kevinzg/facebook-scraper#595]
When I fetch a few posts from a Facebook page (w/ 0.2.50), it seems the results are different from what I see when logged into Facebook and viewing the same page in a browser. 

When I view the same page, while not logged in, it seems to match what the scraper retrieves. 

Facebook seems to be showing different content based on whether one is logged in. 

Oddly, more recently updated content appears to be shown to unauthenticated users and thus the scraper. 

I just noticed this, is this new? Any thoughts? I'm not sure how to deal with, it seems beneficial to use the latest posts in my application - but I'm not sure what's going on entirely.

**Comment by neon-ninja** [comment]
Yes, Facebook shows you different content if you're logged in. Things like post order are personalised to your account, posts most relevant to you are shown first. This is not new. If you want the scraper to be logged in too, you can pass cookies as per the readme.
